### ISSUE TITLE
Update MongoDB driver dependency

### ISSUE THREAD

**Issue by ctrlsam** [pkosiec/mongo-seeding#187]
The [closeConnection](https://github.com/pkosiec/mongo-seeding/blob/a0a46982cb71a0f54db4e0b90e41c5f6f0ccc29f/core/src/database/database.ts#L103) function needs to be reworked to check if the db is connected, without using the deprecated method `isConnected`

```
(node:169094) DeprecationWarning: isConnected is deprecated and will be removed in the next major version
    at Database.<anonymous> (/home/user/Programming/some_project/node_services/node_modules/mongo-seeding/src/database/database.ts:105:38)
    at Generator.next (<anonymous>)
    at /home/user/Programming/some_project/node_services/node_modules/mongo-seeding/dist/database/database.js:8:71
    at new Promise (<anonymous>)
    at Object.<anonymous>.__awaiter (/home/user/Programming/some_project/node_services/node_modules/mongo-seeding/dist/database/database.js:4:12)
    at Database.closeConnection (/home/user/Programming/some_project/node_services/node_modules/mongo-seeding/dist/database/database.js:94:16)
    at Seeder.<anonymous> (/home/user/Programming/some_project/node_services/node_modules/mongo-seeding/src/index.ts:144:24)
    at Generator.next (<anonymous>)
    at fulfilled (/home/user/Programming/some_project/node_services/node_modules/mongo-seeding/dist/index.js:15:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by pkosiec** [comment]
Hi @ctrlsam, thank you for reporting this. 

I will rename this issue to "Update MongoDB driver dependency" and try to address that in Mongo Seeding 4. Cheers!

**Comment by pkosiec** [comment]
Finally resolved in #203 🥳
